requirements/bootstrap-lock.txt

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
I'm wondering if it makes sense to rework the naming conventions of these lock files. Mainly it is sort of hard to follow which is used where. What are your thoughts on this (I'm up for a different naming)?

* `requirements/pip.txt` - A ranged dependency on versions of pip that are required to build/install the CLI.
* `requirements/build-exe.txt` - A ranged dependency on versions of `pyinstaller` allowed
* `requirements/download-deps/bootstrap-extras.txt`- Exact pinned versions of extra dependencies that may be used as part of the `--with-downloaded-deps` install process for building/installing any of our dependencies.
* `requirements/download-deps/bootstrap-lock.txt` - Lock consisting of requirements from `pip.txt`, `bootstrap-extras.txt`, and build reqs from `pyproject.toml`
* `requirements/download-deps/system-sandbox-lock.txt` - Lock consisting of runtime dependencies from `pyproject.toml`
* `requirements/download-deps/portable-exe-lock.txt` - Lock consisting of runtime dependencies from `pyproject.toml` and `build-exe.txt`.

This feels a little more easy to understand which files are used in which portions of the build/install process.